spotlight-client/src/contentApi/sources/us_pa.ts

```diff
@@ -660,8 +661,9 @@ const content: TenantContent = {
         { id: "QUE", label: "Quehanna Boot Camp" },
         { id: "ROC", label: "Rockview" },
         { id: "SMI", label: "Smithfield" },
+        { id: "SMR", label: "Somerset" },
         { id: "WAM", label: "Waymart" },
-        { id: "OTHER", label: "Other Facilities" },
+        { id: "OTHER", label: "Other" },
```

Comment:
I changed this from "Other Facilities" to "Other" since if someone is furloughed they aren't in a facility? thoughts?